packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -189,6 +198,14 @@ export const HelpCenterContactForm = () => {
 		switch ( mode ) {
 			case 'CHAT': {
 				if ( supportSite ) {
+					recordTracksEvent( 'calypso_inlinehelp_contact_submit', {
+						support_variation: 'happychat',
+					} );
+
+					recordTracksEvent( 'calypso_help_live_chat_begin', {
+						site_plan_product_id: supportSite ? supportSite.plan?.product_id : null,
```

Comment:
Seems like `supportSite` is already `true` (see the `if` a few lines above), so this check should not be necessary.

Reply:
Totally right, thanks for noticing!